datafusion/sqllogictest/test_files/window.slt

```diff
@@ -3929,7 +3929,11 @@ b 1 3
 a 1 4
 b 5 5
 
-statement error DataFusion error: Error during planning: Projection references non-aggregate values: Expression aggregate_test_100.c1 could not be resolved from available columns: rn
+# expected failed message:
+#         DataFusion error: expand_wildcard_rule
+#  caused by
+#  Schema error: No field named aggregate_test_100.c1. Valid fields are rn.
+statement error
```

Comment:
Is this TODO?

Reply:
No, it's the actual error message we threw. This pattern is hard to match, so I left a comment to show what it looks like. I think I can just remove it, as it might confuse someone. 🤔

Reply:
I think leaving `Schema error: No field named aggregate_test_100.c1. Valid fields are rn.` is good enough

Reply:
Sounds great.